src/ra_server.erl

```diff
@@ -1485,11 +1483,17 @@ handle_follower(#append_entries_reply{}, State) ->
 handle_follower(election_timeout,
                 #{cfg := #cfg{log_id = LogId},
                   membership := Membership} = State) when Membership =/= voter ->
-    ?DEBUG("~ts: follower ignored election_timeout, non-voter: ~p0",
+    ?DEBUG("~ts: follower ignored election_timeout, non-voter: ~0p",
            [LogId, Membership]),
     {follower, State, []};
 handle_follower(election_timeout, State) ->
     call_for_election(pre_vote, State);
+handle_follower(try_become_leader,
+                #{cfg := #cfg{log_id = LogId},
+                  membership := Membership} = State) when Membership =/= voter ->
+    ?INFO("~ts: follower ignored try_become_leader, non-voter: ~0p",
```

Comment:
Should this be a `debug` message or is the idea to use this level temporarily during the review phase?

Reply:
I intentionally went with an `info`, because this scenario should only happen after an explicit action (such as `rabbitmq-queues rebalance` in case of RMQ). Since the member refuses to do what it was told, I think some feedback is in order.

Reply:
this would happen very rarely and would provide context as to why a rebalance didn't complete for example